primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -332,35 +334,81 @@ impl<E: Pairing> UnivariatePCS for UnivariateKzgPCS<E> {
         num_points: usize,
         domain: &Radix2EvaluationDomain<Self::Evaluation>,
     ) -> Result<Vec<Self::Proof>, PCSError> {
-        #[cfg(feature = "naive-kzg-multi-open")]
+        // #[cfg(feature = "naive-kzg-multi-open")]
```

Comment:
@ggutoski Do we still want to keep the naive code here?

Reply:
I see no reason to keep it.

Reply:
Removed in c64a5bd3b83d19b5344611a0f74d9906b542163a